web/src/lib.rs

```diff
@@ -345,6 +348,7 @@ impl Ruffle {
             key_up_callback: None,
             timestamp: None,
             has_focus: false,
+            https_context: protocol == "https:",
```

Comment:
This isn't hooked up yet, but it's idea for where we could put the https context flag, and then I guess consume it in the player somehow.